src/common.rs

```diff
@@ -23,6 +23,7 @@ pub enum Architecture {
     S390x,
     Sparc64,
     Wasm32,
+    LoongArch64,
```

Comment:
This enum is (mostly) sorted. Can you move this variant to the appropriate location?

Reply:
Done.

Reply:
Sorry, forgot to sort the `match` expression in `address_size`.  Fixed now.